.github/workflows/test.yml

```
@@ -170,15 +155,12 @@
 
       - name: Set up Xcode version
         run: sudo xcode-select -s /Applications/Xcode_16.1.app/Contents/Developer
-      
-      - run: bun install
-        working-directory: example
-      
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)